.github/workflows/tests.yml

```diff
@@ -40,14 +44,14 @@ jobs:
       - name: Build
         run: ./bin/unipoints build
       - name: Check built assets
-        if: startsWith(matrix.os, 'windows') == false
+        if: ${{ startsWith(matrix.os, 'windows') == false }}
```

Comment:
This is Explicit Viktor running at half steam.
Full throttle would be quoting all YAML strings.

Reply:
I find quoted strings somehow less readable

Reply:
Yes, they are less readable.
But YAML is for machines, not for hoomans :)
So I do everything to make is safe.

Reply:
Also, in `if` cases the `${{ ... }}` are optional, right?
When there are two alternatives, I always prefer the less verbose syntax (like `[]` vs `arrray()` in PHP).

Reply:
Yaml is read both by humans and machines (I wrote it, not chatgpt 😉 )

Reply:
> I always prefer the less verbose syntax

Okay-okay I cannot really talk to developers. My single goal is long error-free years.

Reply:
I let you finish/close/merge this PR.

Reply:
I haven't come up with 4-space indentation and non-indented lists!
Those take months to get used to.

Reply:
I have 103 0️⃣0️⃣0️⃣ 0️⃣0️⃣0️⃣ reasons.

![image](https://github.com/mlocati/unipoints/assets/952007/86377158-c892-4e64-aa25-a32202bc4c37)